### ISSUE TITLE
Regex doesn't match necessary folders

### ISSUE THREAD

**Issue by adambrgmn** [sjofartstidningen/sst-indesign-export#1]
The regex in `getDocumentData` does not match all necessary folders.

Currently it's:
```js
const re = /\/(\d{4})\/(\d{2})$/;
//             ^        ^
//           / 2018   / 01
const [, year, issue] = re.exec(doc.filePath);
```

This will match only the following structure: `/path/to/folder/2017/01` (where 2017 can be any four digit year, and 01 can be any two digit issue number).

But at Sjöfartstidningen we might have special issues tied to a certain ”normal” issue, e.g ”02-Tanker Special”.

Therefore the regex should be:
```js
const re = /\/(\d{4})\/(\d{2}(?:\w|\s|-)*)$/; // also skip the g flag as only one match is needed per input
//             ^        ^       ^
//           / 2017   / 01      Hell0-wor1d
const [, year, issue] = re.exec(doc.filePath);
```

This will match all strings below:
`/path/to/folder/2017/01`
`/path/to/folder/2017/01-Special Edition`
`/path/to/folder/2017/02`
`/path/to/folder/2017/02-Hello-world`

**In short, the issue part of the folder structure does not have to be two digits.**
